zap/src/main/java/org/zaproxy/zap/extension/spider/ExtensionSpider.java

```diff
@@ -57,7 +56,12 @@
 import org.zaproxy.zap.users.User;
 import org.zaproxy.zap.view.ZapMenuItem;
 
-/** The ExtensionSpider is the Extension that controls the Spider. */
+/**
+ * The ExtensionSpider is the Extension that controls the Spider.
+ *
+ * @deprecated (2.12.0) See the spider add-on in zap-extensions instead.
+ */
+@Deprecated
```

Comment:
Should we be using the `@Deprecated(since = "2.12.0", forRemoval = true)` form?

Reply:
That's not available in Java 8.

Reply:
Oh okay